File: codeforces/round548/A/AdHocCounting/test/SolutionTests.java
import static org.junit.Assert.*;

import java.io.*;

import org.junit.*;

public class SolutionTests {

    @Test
    public void test0() throws FileNotFoundException {
        InputReader in = new InputReader(new FileInputStream("test_data/0.in"));
        Solution solution = new Solution(in);
        int result = solution.solve();
        assertEquals(6, result);
    }

    @Test
    public void test1() throws FileNotFoundException {
        InputReader in = new InputReader(new FileInputStream("test_data/1.in"));
        Solution solution = new Solution(in);
        int result = solution.solve();
        assertEquals(10, result);
    }

}

File: withgoogle/KickStart/2020/Round A/Workout/BinSearch/Solution.Tests/SolutionTests.cs
using System;
using System.IO;
using NUnit.Framework;

public class SolutionTests {
	[Test]
	public void Test0() {
		Solution solution = new Solution(new InputReader(new StreamReader("../../../../test_data/0.in")));
		string[] result = solution.Solve();
		Console.WriteLine("Here");
		Assert.AreEqual(new string[] {
			"Case #1: 50"
		}, result);
	}

	[Test]
	public void Test1() {
		Solution solution = new Solution(new InputReader(new StreamReader("../../../../test_data/1.in")));
		string[] result = solution.Solve();
		Console.WriteLine("Here");
		Assert.AreEqual(new string[] {
			"Case #1: 2",
			"Case #2: 3",
			"Case #3: 1"
		}, result);
	}
}
